### PR TITLE
Https

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**
## Type
- Enhancement

## Pull request description
Add https entry in README

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -69,6 +69,8 @@ deny from env=stayout
 
 	# redirect all trafic to https
 	# Header set Strict-Transport-Security "max-age=31536000; includeSubDomains" env=HTTPS
+	# Header always edit Set-Cookie (.*) "$1; HTTPOnly; Secure"
+	# Header set Strict-Transport-Security: "max-age=63072000"
```
Comment (carakas) [inline .htaccess]: Doesn't this conflict with `Header set Strict-Transport-Security "max-age=31536000; includeSubDomains" env=HTTPS`at line 71?

File: .htaccess [hunk 2]
```diff
@@ -69,6 +69,8 @@ deny from env=stayout
 
 	# redirect all trafic to https
 	# Header set Strict-Transport-Security "max-age=31536000; includeSubDomains" env=HTTPS
+	# Header always edit Set-Cookie (.*) "$1; HTTPOnly; Secure"
```
Comment (carakas) [inline .htaccess]: This is the default when you set cookies with fork, but you can choose other options there, won't this prevent people from setting the cookies the way they want?

File: .htaccess (hunk 1 above)
Comment (jonasdekeukelaere) [inline .htaccess]: My mistake, did a check for the first addition, never checked for the second one.

Comment (jonasdekeukelaere) [inline .htaccess]: Fixed 2726cdd43

File: .htaccess (hunk 2 above)
Comment (jonasdekeukelaere) [inline .htaccess]: Removed it.

**Pull request merged**